libs/utils/utils.js

```diff
@@ -654,6 +654,16 @@ export function loadPrivacy() {
   });
 }
 
+async function loadJarvisChat() {
+  const config = getConfig();
+  const jarvis = getMetadata('jarvis-chat');
+  if (!config.jarvis?.id || !config.jarvis?.version) return;
+  if (jarvis === 'on') {
+    const { initJarvisChat } = await import('../features/jarvis-chat.js');
+    initJarvisChat(config, loadScript, loadStyle);
+  }
+}
```

Comment:
I get that this is delayed, but is there any reason to _not_ load this out of footer instead?

We're trying to reduce the footprint of utils since it is needed to be fully downloaded to get to LCP, and ideally we would get this out.

The alternative is to use this as an opportunity to remove loadDelayed out of utils completely because it shouldn't be here anyway.

Reply:
Jarvis chat can also be used on pages without header or footer; in the Milo world we would not be able to load Jarvis if the loading happens within the footer.

I do agree with the second approach, it's something that I previously mentioned in the comments, but we also want to tackle the Privacy component while we're at it and make some test. This will require additional effort that we're willing to do in an upcoming PR, scheduled for June.

Reply:
> Jarvis chat can also be used on pages without header or footer; in the Milo world we would not be able to load Jarvis if the loading happens within the footer.
💯 

Remove loadDelayed: https://jira.corp.adobe.com/browse/MWPW-131408